galleries/examples/shapes_and_collections/ellipse_arrow.py

```diff
@@ -0,0 +1,46 @@
+"""
+============
+Ellipse with arrow Demo
+============
```

Comment:
These lines of `=` need to be the same length as the title.  I think this is why the documentation build check is failing.

Reply:
yes, of course

Reply:
(There is one other doc build failure, but it isn't on this PR, xref #25782)